server/server.ts

```diff
@@ -60,6 +61,7 @@ if (process.env.NODE_ENV === 'production') {
 	app.use(Sentry.Handlers.requestHandler({ user: ['id', 'slug'] }));
 	app.use(enforce.HTTPS({ trustProtoHeader: true }));
 }
+app.use(deduplicateSlash());
```

Comment:
The main fix. This middleware strips repeating slash characters from request paths.